package.json

```diff
@@ -88,22 +91,22 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.6.0",
-    "@storybook/addon-actions": "^5.1.11",
-    "@storybook/addon-links": "^5.1.11",
-    "@storybook/addons": "^5.1.11",
-    "@storybook/react": "^5.1.11",
+    "@babel/core": "^7.12.10",
```

Comment:
I see there are many packages got upgraded. Can you please make sure that the functionality is in tact with the latest versions across these packages.